analytics/models.py

```diff
@@ -0,0 +1,39 @@
+from django.db import models
+
+from zerver.models import Realm, UserProfile
+from zerver.lib.str_utils import ModelReprMixin
+
+# only would ever touch by hand
+class Anomaly(ModelReprMixin, models.Model):
+    info = models.CharField(max_length=1000) # type: text_type
```

Comment:
This is intended to just be textual descriptions of what's up with weird things?